include/openenclave/edger8r/common.h

```diff
@@ -66,20 +66,33 @@ OE_INLINE oe_result_t oe_add_size(size_t* total, size_t size)
     return result;
 }
 
-#define OE_ADD_SIZE(total, size)                                   \
+#define OE_COMPUTE_SIZE(a, b, c)                 \
```

Comment:
It's not obvious from the signature that a and b are inputs and c is an output.
Consider: Can we either
a) somehow turn this into an inline function where it's more obvious, or 
b) somehow have the macro "return" of c, so you can do `c = OE_COMPUTE_SIZE(a, b);`
?

Reply:
The update should address this. Thanks!